I retested and it works. Specifying `--verbose` shows the error printout. (In non-verbose mode, only the sdformat error message is printed, `Error [Param.cc:452] Unable to set value [-nan -nan -nan] for key[size]`.)

Linux CI looks okay.

@j-rivero @scpeters Mac is saying pip3 not found?
```
++ pip3 install lxml
./scripts/jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash: line 81: pip3: command not found
Build step 'Execute shell' marked build as failure
Recording test results
ERROR: Step ‘Publish JUnit test result report’ failed: Test reports were found but none of them are new. Did leafNodes run? 
For example, /Users/jenkins/workspace/gazebo-ci-pr_any-homebrew-amd64/build/test_results/EXAMPLE_examples_build.xml is 1 day 15 hr old
```

and Windows is not finding Qt:
```
CMake Error at C:/Program Files/CMake/share/cmake-3.15/Modules/FindQt4.cmake:1314 (message):
  Found unsuitable Qt version "" from NOTFOUND, this code requires Qt 4.x
```

None of this is related to this PR, but I don't know if I should approve and merge if some of the CIs hadn't run...

--- a/gazebo/rendering/Visual.cc
+++ b/gazebo/rendering/Visual.cc
@@ -799,7 +799,9 @@ void Visual::SetScale(const math::Vector3 &_scale)
   {
     this->dataPtr->sceneNode->setScale(
         Conversions::Convert(this->dataPtr->scale));
-  } else {
+  }
+  else
+  {
     gzerr << GetName() << ": Size of the collision contains one or several zeros." <<
       " Collisions may not visualize properly." << std::endl;
   }

